ReactAndroid/src/main/java/com/facebook/react/v8executor/Android.mk

```diff
@@ -12,6 +12,7 @@ LOCAL_MODULE := v8executor
 LOCAL_SRC_FILES := $(wildcard $(LOCAL_PATH)/*.cpp)
 
 LOCAL_C_INCLUDES := $(LOCAL_PATH)
+LOCAL_C_INCLUDES += $(THIRD_PARTY_NDK_SRC_DIR)/..
```

Comment:
can we in above like only

LOCAL_C_INCLUDES := $(LOCAL_PATH)  $(THIRD_PARTY_NDK_SRC_DIR)/..